packages/kit/src/vite/index.js

```diff
@@ -61,7 +62,54 @@ const enforced_config = {
  * @return {import('vite').Plugin[]}
  */
 export function sveltekit() {
-	return [...svelte(), kit()];
+	return [...svelte(), env(), kit()];
+}
+
+/**
+ * Returns the SvelteKit environment plugin. Replaces `$env/static/*` imports
+ * with the actual environment variables.
+ *
+ * @return {import('vite').Plugin}
+ */
+function env() {
+	/** @type {import('types').ValidatedConfig} */
+	let svelte_config;
+
+	/** @type {import('vite').ConfigEnv} */
+	let vite_config_env;
+
+	return {
+		name: 'vite-plugin-env',
```

Comment:
I think `vite-plugin-svelte` uses a convention of `:subplugin`, which seems nice.
```suggestion
		name: 'vite-plugin-svelte-kit:env',
```